example_python_travis.yml

```diff
@@ -13,29 +14,32 @@ addons:
 
 env:
   global:
+    # Doctr deploy key for <org>/<repo>
```

Comment:
Maybe in the template we should be verbose such that it's obvious what needs to be done. I had trouble on-boarding someone new to doctr, and either steps here or a link to a document could save us and the end users frustration.

Reply:
I've never gone through the process myself so I'm not sure what to write. For now, I've left a link to the doctr documentation in the README but that's not very helpful. Could one of you add more info either here or in the README?

Reply:
OK, no problem - created a new issue for it.